src/robusta/core/playbooks/crash_reporter.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from typing import Optional
+
+from robusta.core.model.base_params import NamedRegexPattern
+from robusta.core.playbooks.pod_utils.crashloop_utils import get_crash_report_enrichments
+from robusta.core.reporting import Finding, FindingSource, FindingSeverity, FileBlock
+from robusta.core.reporting.base import EnrichmentType
+from robusta.core.reporting.finding_subjects import PodFindingSubject
+from robusta.integrations.kubernetes.autogenerated.events import PodEvent
+from robusta.integrations.kubernetes.custom_models import RegexReplacementStyle
+
+
+def send_crash_report(
+        event: PodEvent,
+        action_name: str,
+        regex_replacer_patterns: Optional[NamedRegexPattern] = None,
+        regex_replacement_style: Optional[RegexReplacementStyle] = None,
+):
+    pod = event.get_pod()
+
+    all_statuses = pod.status.containerStatuses + pod.status.initContainerStatuses
+    crashed_container_statuses = [
+        container_status
+        for container_status in all_statuses
+        if container_status.state.waiting is not None and container_status.restartCount >= 1
+    ]
+
+    finding = Finding(
+        title=f"Crashing pod {pod.metadata.name} in namespace {pod.metadata.namespace}",
+        source=FindingSource.KUBERNETES_API_SERVER,
+        severity=FindingSeverity.HIGH,
+        aggregation_key=action_name,
+        subject=PodFindingSubject(pod),
+    )
+
+    enrichments = get_crash_report_enrichments(pod)
+    if enrichments:
+        for enrichment in enrichments:
+            finding.add_enrichment(enrichment.blocks,
+                                   enrichment_type=enrichment.enrichment_type,
+                                   title=enrichment.title)
+
+    for container_status in crashed_container_statuses:
+        try:
+            container_log = pod.get_logs(
+                container_status.name,
+                previous=True,
+                regex_replacer_patterns=regex_replacer_patterns,
+                regex_replacement_style=regex_replacement_style,
+            )
+            metadata = None
+            if not container_log:
+                metadata = {
```

Comment:
Is this for the UI? To present an empty log?
How does this look in Slack and the other Sinks?

Reply:
Slack and msteams sinks screenshots:

![Screenshot 2024-02-21 at 5 35 05 PM](https://github.com/robusta-dev/robusta/assets/182092/2298dfb1-2646-4de6-86dc-9e0b2d193d03)


<img width="859" alt="Screenshot 2024-02-21 at 5 49 15 PM" src="https://github.com/robusta-dev/robusta/assets/182092/8e68cf33-a1c3-4ee6-b5fb-6fd5b7ac93a1">

Reply:
How come we don't see the empty file on Slack?

On any event, is there a way to send the empty log only to the UI sink?
Maybe we can create a new type of block, `EmptyFileBlock` that is handled only in the UI sink? (and sends it as an empty file)

Or other solutions that will have a similar result?